packages/webpack-cli/lib/plugins/CLIPlugin.js

```diff
@@ -20,7 +20,7 @@ class CLIPlugin {
         new PrefetchPlugin(null, this.options.prefetch).apply(compiler);
     }
 
-    async setupBundleAnalyzerPlugin(compiler) {
+    setupBundleAnalyzerPlugin(compiler) {
         // eslint-disable-next-line node/no-extraneous-require
         const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
```

Comment:
It will be `import` in future, because packages start to migrate on ES modules, so better to keep `async` here

Reply:
get